nautobot/core/templates/inc/computed_fields_panel.html

```diff
@@ -0,0 +1,13 @@
+{% load computed_fields %}
+
+
+{% if object.has_computed_fields %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>Computed Fields</strong>
```

Comment:
I think we would make sense to combine `Computed Fields` and `Custom Fields` panels in the detail view under a single `Additional Fields` panel.
It feels like we are exposing too much of the internal logic to the user.
I understand this might not be addressed in this PR but I think we should open a new issue to keep track of that
cc @jedelman8